<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTransformFusionModule</class>
 <widget class="qSlicerWidget" name="qSlicerTransformFusionModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="parameterNodeBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parameter Set</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="ParameterComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTransformVisualizerNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ioBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Input/Output</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="inputTransformsBox">
        <property name="title">
         <string>Input Transforms</string>
        </property>
        <layout class="QGridLayout" name="gridlayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="addTransformComboBox">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLLinearTransformNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>false</bool>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="addTransformButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="removeTransformButton">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QListWidget" name="inputTransformList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Output Transform</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="qMRMLNodeComboBox" name="outputTransformComboBox">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLLinearTransformNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>false</bool>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="controlBox_2">
     <property name="text">
      <string>Control</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ctkCollapsibleGroupBox" name="SingleUpdateBox">
        <property name="title">
         <string>Single Update</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="AutoUpdateBox">
        <property name="title">
         <string>Auto Update</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="UpdateRateLabel">
           <property name="text">
            <string>Update Rate (per second)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkDoubleSpinBox" name="updateRateBox">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
           <property name="value">
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="startUpdateButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="stopUpdateButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTransformFusionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputTransformComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTransformFusionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>addTransformComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTransformFusionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParameterComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>735</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
